### PR TITLE
Update xfinity.com.xml

### PR DESCRIPTION
**Pull request by koops76**
#11114

### CODE REVIEW

File: src/chrome/content/rules/xfinity.com.xml [hunk 1]
```diff
@@ -92,47 +56,21 @@ Fetch error: http://cdn.wcdc.xfinity.com/ => https://cdn.wcdc.xfinity.com/: (6,
 	<target host="secure.xfinity.com" />
 	<target host="securestore.xfinity.com" />
 	<target host="customer.stg.xfinity.com" />
-	<!--target host="www.stg.xfinity.com" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/xfinity.com.xml]: WFM

File: src/chrome/content/rules/xfinity.com.xml [hunk 2]
```diff
@@ -23,7 +16,6 @@ Fetch error: http://cdn.wcdc.xfinity.com/ => https://cdn.wcdc.xfinity.com/: (6,
 	ᵈ Dropped
 	ʳ Refused
 
-
 	Problematic hosts in *xfinity.com:
 
 		- constantguard ᶜ
```
Comment (J0WI) [inline src/chrome/content/rules/xfinity.com.xml]: I got a timeout

File: src/chrome/content/rules/xfinity.com.xml [hunk 3]
```diff
@@ -1,15 +1,8 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://cdn.xfinity.com/ => https://cdn.xfinity.com/: (6, 'Could not resolve host: cdn.xfinity.com')
-Fetch error: http://cdn.ch2.xfinity.com/ => https://cdn.ch2.xfinity.com/: (6, 'Could not resolve host: cdn.ch2.xfinity.com')
-Fetch error: http://cdn.wcdc.xfinity.com/ => https://cdn.wcdc.xfinity.com/: (6, 'Could not resolve host: cdn.wcdc.xfinity.com')
-
 	For rules causing false/broken MCB, see xfinity.com-falsemixed.xml.
```
Comment (J0WI) [inline src/chrome/content/rules/xfinity.com.xml]: Obsolete, see #11358

File: src/chrome/content/rules/xfinity.com.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/xfinity.com.xml]: @J0WI Done.

File: src/chrome/content/rules/xfinity.com.xml (hunk 2 above)
Comment (koops76) [inline src/chrome/content/rules/xfinity.com.xml]: @J0WI Done.

File: src/chrome/content/rules/xfinity.com.xml (hunk 3 above)
Comment (koops76) [inline src/chrome/content/rules/xfinity.com.xml]: @J0WI Done.

File: src/chrome/content/rules/xfinity.com.xml [hunk 4]
```diff
@@ -18,18 +17,16 @@
...
 
-		- constantguard ᶜ
-		- my ᴬ
-		- serviceo ᵐ
-		- www.stg ˣ
-		- store ²
-		- tvgo ᴬ
+		- my ²
+		- serviceo ³
+		- www.stg ⁴
```
Comment (J0WI) [inline src/chrome/content/rules/xfinity.com.xml]: That comment is also obsolete.

Comment (koops76) [inline src/chrome/content/rules/xfinity.com.xml]: @J0WI Done.

**Pull request merged**